app.py

```
@@ -388,7 +389,8 @@ def load_chat(chat_id):
         (chat_id,),
     ).fetchall()
 
-    return jsonify({"messages": [dict(msg) for msg in messages]})
+    context = Chat.get_context(chat_id)
```

Ruff (0.8.2)

Undefined name `Chat`

(F821)